lib/Analysis/Utility.cpp

```diff
@@ -437,6 +437,8 @@ bool supportMFMA(triton::DotOp op) {
 
   auto aShape = aTy.getShape();
   auto bShape = bTy.getShape();
+  if (aShape.size() != 2 || bShape.size() != 2)
```

Comment:
Does it mean MFMA doesn't support 3d?

Reply:
Is `supportMFMA` even used anywhere ? Maybe I added this by mistake - will check

Reply:
This is for AMD GPU, we probably don't want to modify it unless we test it (and probably at this stage we shouldn't)

Reply:
Yea I don't know why i changed this, probably by mistake, will revert.